Libraries/Components/Touchable/TouchableNativeFeedback.android.js

```diff
@@ -13,6 +14,7 @@ const Platform = require('Platform');
 const React = require('React');
 const PropTypes = require('prop-types');
 const ReactNative = require('ReactNative');
+// $FlowFixMe
```

Comment:
This is to fix the flow warning about not importing from untyped files, right? If that is the case then lets just not turn on strict-local in this file.

Reply:
Thank you for reviewing.
I applied flow to `Touchable.js` then I removed `FlowFixMe` from `TouchableNativeFeedback.android.js`.